custom/icds_reports/ucr/data_sources/dashboard/daily_feeding_forms.json

```diff
@@ -86,6 +86,32 @@
         },
         "create_index": true
       },
+      {
+        "column_id": "supervisor_id",
+        "datatype": "string",
+        "type": "expression",
+        "expression": {
+          "type": "ancestor_location",
+          "location_id": {
```

Comment:
`location_id` can be simplified to: 
```
"location_id": {
  "type": "named",
  "name": "user_location_id"  # this already exists in the doc as a named expression
}
```

The same can be done for `state_id`

Reply:
👍